app/assets/javascripts/models/beta/task.js

```diff
@@ -0,0 +1,69 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+
+export function newTask(projectId, storyId, task) {
+  let newTask = {
+    name: task,
+    done: false
+  }
+  return httpService
+    .post(`/projects/${projectId}/stories/${storyId}/tasks`, newTask)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch(function (error) {
+      console.log(error);
+    });
+};
+
+export function updateTaskStatus(projectId, storyId, taskId, status) {
+  const taskStatus = changeCase.snakeKeys(status);
+  return httpService
+    .put(`/projects/${projectId}/stories/${storyId}/tasks/${taskId}/`, taskStatus)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch(function (error) {
+      console.log(error);
+    });
+};
+
+export function excludeTask(projectId, storyId, taskId) {
```

Comment:
Change this to:

```suggestion
export const excludeTask = (projectId, storyId, taskId) => {
```
to be equal to the other ones.